devs/lib/clientcmds.ts

```diff
@@ -371,15 +371,57 @@ Array.prototype.reduce = function (callbackfn: any, initialValue: any) {
 
 declare module "@devicescript/core" {
     interface I2C {
+        /**
+         * 
+         * @param devAddr is a 7 bit i2c address
```

Comment:
typically `name` and the rest of the sentence may be split so it is likely that the "is " will read funny.